knowledge-base/using-the-html5--viewer-in-mvc-applications-and-binding-data.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Using the HTML5 Viewer in MVC applications and binding data
+description: This article explains how to use the  the HTML5 Viewer in MVC applications and binding data.
+type: how-to
+page_title: How to Use the HTML5 Viewer in MVC Applications and Binding Data
+slug: using-the-html5--viewer-in-mvc-applications-and-binding-data
+position: 
+tags: 
+ticketid: 1532522
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
```

Comment:
Why don't you add more stuff here? Like Viewer:HTML 5, Technology, or Project Type:MVC